client/js/keybinds.js

```diff
@@ -107,6 +107,17 @@ Mousetrap.bind(["alt+a"], function (e) {
 	return false;
 });
 
+// Show the help menu
+Mousetrap.bind(["command+/", "ctrl+/"], function (e) {
```

Comment:
All other navigation keys are Alt based ones, why does this use control?

Meaning Alt-/ would be more in line with what we currently have

Reply:
The original issue (#4286) suggested cmd/ctrl, alt definitely fits in better.